src/views/AnnualReport.vue

```diff
@@ -115,14 +126,15 @@
                     <p v-else>This is your list of directors active as of {{asOfDate}}, including
                       directors that were ceased at a later date.</p>
                   </header>
-                  <directors ref="directorsList"
+                  <directors
+                    ref="directorsComponent"
+                    :directors.sync="updatedDirectors"
+                    :componentEnabled="allowChange('cod')"
+                    @original="originalDirectors=$event"
```

Comment:
Note the new interface here as well.

And, this interface is now used consistently between all filings.